sql/char_fishing.sql

```diff
@@ -0,0 +1,22 @@
+-- --------------------------------------------------------
+-- Add history for player fishing data
+-- --------------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS `char_fishing` (
```

Comment:
As I'm looking at this more (and I still don't know how any part of fishing works), this is denormalised right? We'll only ever have one row per char which we read, update, and then write back to the db with their best results etc.? 

We absolutely cannot have a forever history of  every single entry of every fish of every char that has been submitted to every contest etc. - because we'll then have to search over it and having that historical data isn't worth the cost (like it is for AH, which we have utils to truncate)

Reply:
That is correct.  At the start of every contest, the entry table is wiped clean, and the only thing that gets saved is the number of times a player has achieved either rank 1, rank 2, rank 3, or rank 4-10.  One row per char only.,